modules/querier/http.go

```diff
@@ -442,6 +443,16 @@ func handleError(w http.ResponseWriter, err error) {
 }
 
 func writeFormattedContentForRequest(w http.ResponseWriter, req *http.Request, m proto.Message, span oteltrace.Span) {
+	// Check for both explicit nil and typed nil pointers (e.g., (*tempopb.SearchResponse)(nil))
+	// A typed nil pointer is not equal to nil when passed as an interface, so we need reflection
+	if m == nil || (reflect.ValueOf(m).Kind() == reflect.Ptr && reflect.ValueOf(m).IsNil()) {
```

Comment:
wut

Reply:
Nil and interface checking is hard because an interface is a type and value so you have to check both of them. Regardless this was a bandaid fix and I think I fixed the root cause in livestore.